### ISSUE TITLE
Error when specify multiple serviceAccountKeyFile on kube-apiserver

### ISSUE THREAD

**Issue by hatappi** [kubernetes/kops#7645]
**1. What `kops` version are you running? The command `kops version`, will display
 this information.**

Version 1.13.0 (git-be5fb9019)

**2. What Kubernetes version are you running? `kubectl version` will print the
 version if a cluster is running or provide the Kubernetes version specified as
 a `kops` flag.**

v1.15.4

**3. What cloud provider are you using?**

AWS

**4. What commands did you run?  What is the simplest way to reproduce this issue?**

Specify multiple files for `serviceAccountKeyFile`

```yaml
spec:
  fileAssets:
    - name: service-account-private-key-file
      path: /srv/kubernetes/assets/service-account-private-key
      isBase64: true
      content: {base64 encoded private key}
    - name: service-account-key-file
  kubeAPIServer:
    serviceAccountKeyFile:
      - "/srv/kubernetes/server.key"
      - "/srv/kubernetes/assets/service-account-signing-key"
```

run `kops rolling-update cluster` .

**5. What happened after the commands executed?**
There was an error in the kube-apiserver log

```bash
 error: invalid authentication config: open /srv/kubernetes/server.key,/srv/kubernetes/assets/service-account-signing-key: no such file or directory
```

**6. What did you expect to happen?**

No error occurs even if multiple service account key files are specified

**7. Please provide your cluster manifest. Execute
  `kops get --name my.example.com -o yaml` to display your cluster manifest.
  You may want to remove your cluster name and other sensitive information.**

```yaml
apiVersion: kops/v1alpha2
kind: Cluster
metadata:
  creationTimestamp: 2019-05-05T02:42:37Z
  generation: 1
spec:
  api:
    loadBalancer:
      type: Public
  authorization:
    rbac: {}
  channel: stable
  cloudProvider: aws
  etcdClusters:
  - etcdMembers:
    - instanceGroup: master-ap-northeast-1a
      name: a
    name: main
  - etcdMembers:
    - instanceGroup: master-ap-northeast-1a
      name: a
    name: events
  iam:
    allowContainerRegistry: true
    legacy: false
  kubeAPIServer:
    serviceAccountKeyFile:
    - /srv/kubernetes/server.key
    - /srv/kubernetes/assets/service-account-signing-key
  kubelet:
    anonymousAuth: false
    authenticationTokenWebhook: true
  kubernetesVersion: 1.15.4
  networking:
    amazonvpc: {}
  topology:
    dns:
      type: Public
    masters: private
    nodes: private
```

**8. Please run the commands with most verbose logging by adding the `-v 10` flag.
  Paste the logs into this report, or in a gist and provide the gist link here.**

N/A

**9. Anything else do we need to know?**

I confirmed manifest file `cat /etc/kubernetes/manifests/kube-apiserver.manifest`.
The following command was executed.

```
/usr/local/bin/kube-apiserver ... --service-account-key-file=/srv/kubernetes/server.key,/srv/kubernetes/assets/service-account-signing-key ...
```

`service-account-key-file` need to specify `stringArray`.
https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/
In other words, need to specify the following command

```
/usr/local/bin/kube-apiserver ... --service-account-key-file=/srv/kubernetes/server.key --service-account-key-file=/srv/kubernetes/assets/service-account-signing-key ...
```

I think [pkg/apis/kops/componentconfig.go](https://github.com/kubernetes/kops/blob/master/pkg/apis/kops/componentconfig.go#L432) needs to be modified

```diff
-       ServiceAccountKeyFile []string `json:"serviceAccountKeyFile,omitempty" flag:"service-account-key-file"`
+       ServiceAccountKeyFile []string `json:"serviceAccountKeyFile,omitempty" flag:"service-account-key-file,repeat"`
```
